battleship_runner.rb

```diff
@@ -4,7 +4,8 @@
 play = Play.new
 
 play.game.start_game
-
+play.game.user_board.set_board_size
+require "pry"; binding.pry
```

Comment:
We'll have to adjust the render and comp_board methods to match this refactor but this is a great blueprint.